src/styles/_catalog.scss

```diff
@@ -1,22 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, $card-width);
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px) and (max-width: 768px) {
```

Comment:
You can omit the combination of the min- and the max-width at the same time. Use only one, next rule will overwrite the previous one.